docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -337,13 +337,35 @@ the environment, and running any activation scripts that the environment may
 contain. These activation scripts are how packages can set arbitrary
 environment variables that may be necessary for their operation.
 
+When `installing Anaconda <http://docs.continuum.io/anaconda/install.html>`_,
+you have the option to “Add Anaconda
+to my PATH environment variable.” This is not recommended because the
+add to PATH option appends Anaconda to path. This means Anaconda’s features
+are found last and the installer appending to PATH does not call the
```

Comment:
It's not that anaconda's features are found last. To be exact, PATH lookup proceeds as follows:

When the user enters a non-relative and non-fullpath command (e.g. `conda` as opposed to `/opt/conda/bin/conda` and as opposed to `../conda/bin/conda`, then the operating system searches for `conda` on `PATH`. This is literally the dumbest lookup you can imaging. PATH is split into an ordered list at `:` characters on Unix and at `;` characters on Windows. Then the OS appends the program name (e.g. `conda`) to each entry on this list in turn. If a file is found to exist at that location then it is executed and the procedure finishes. If it isn't found, then we move onto the next entry doing the same append-then-check procedure. You are right about activate not happening though and that's more important IMHO. Either we respect activate and promote nothing but activate or we get rid of it entirely. In the middle ground lies nothing but frustration and bug reports.